style.css

```diff
@@ -0,0 +1,817 @@
+  color: white;
+  border-radius: 3px;
+}
+
+.button2 {
+  margin-top: 1.5em;
+  margin-bottom: 1.5em;
+  border: none;
+  width: fit-content;
+  align-self: center;
+  background: var(--color-primary);
+  font-weight: 500;
+  font-size: 16px;
+  color: white;
+  border-radius: 3px;
+}
+
+button:hover {
+  box-shadow: 0 8px 16px rgb(54 179 127 / 24%);
+}
+
+button:active {
+  background: #008552;
+}
+
+/* Sections */
+.container {
+  width: 60%;
+  margin: 0 auto;
+}
+
+.container-flex {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.header-nav-desktop {
+  display: none;
+}
+
+.header-nav-phone {
+  top: 0;
+  left: 0;
+  width: 100%;
+  background-color: none;
+  z-index: 100;
+  padding: 0 35px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.menu-btn,
+.nav-menu > button {
+  background-color: white;
+  margin-right: 40px;
+}
+
+.nav-menu > button {
+  margin: 0.5em;
+  align-self: flex-end;
+  cursor: pointer;
+}
+
+.nav-menu > img {
+  width: 2em;
+}
+
+.nav-menu > ul > li {
+  margin-left: 28px;
+  margin-top: 40px;
+  font-size: 24px;
+  font-weight: 600;
+  cursor: pointer;
+}
+
+.nav-menu > ul > li > a {
+  font-size: 24px;
+  font-weight: 600;
+  color: #3a4a42;
+  cursor: pointer;
+}
+
+.link-portfolio:hover,
+.link-about:hover,
+.link-contact:hover {
+  color: var(--color-primary);
+}
+
+.btn-close-nav {
+  position: absolute;
+  font-size: 30px;
+  right: 48px;
+  top: 20px;
+}
+
+.nav-menu {
+  position: fixed;
+  height: 100vh;
+  width: 100%;
+  top: 0;
+  right: -100%;
+  background-color: #fff;
+  backdrop-filter: blur(8px);
+  overflow-x: hidden;
+  overflow-y: hidden;
+  display: flex;
+  flex-direction: column;
+  transition: 0.5s;
+}
+
+.show {
+  left: 0;
+}
+
+/* .menu-btn.hidden {
+  display: none;
+} */
+
+/* .nav-menu.hidden {
+  display: none;
+} */
+
+.header-logo {
+  text-decoration: none;
+  font-family: inter, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  color: #28352f;
+}
+
+.headline {
+  background: #fff url("./images/background.png") no-repeat;
+  background-size: cover;
+  overflow: hidden;
+  background-position: center;
+}
+
+/* .headline h1 {
+  margin-top: 150px;
+} */
+
+.headline p {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
+  margin: 2em 24px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.headline button {
+  background: none;
+}
+
+.social-media {
+  list-style: none;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  width: 170px;
+}
+
+@media (min-width: 992px) {
+  .menu-btn {
+    display: none;
+  }
+
+  .header-logo {
+    display: none;
+  }
+
+  .header-title {
+    padding-top: 3em;
+  }
+
+  .header-nav-desktop {
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-end;
+    padding: 0;
+    padding-top: 1em;
+    margin-left: 500px;
+  }
```

Comment:
- Hey, please, let's try to adjust our styles so the nav can be centered as seen in the Figma t make our design close. see the image below:


*Figma design*
![Screenshot from 2023-01-19 18-06-11](https://user-images.githubusercontent.com/59397159/213512843-0b378bb0-9b1d-4bbb-b35c-642ef3c0b95f.png)

*your design*
![Screenshot from 2023-01-19 18-05-18](https://user-images.githubusercontent.com/59397159/213512850-49587b8b-e66f-4ac1-a9fd-f62db5873faf.png)